src/main/java/uk/gov/companieshouse/api/testdata/model/rest/PublicCompanySpec.java

```diff
@@ -0,0 +1,297 @@
+package uk.gov.companieshouse.api.testdata.model.rest;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import java.util.List;
+
+/**
+ * Requirements a new company must meet for public.
+ */
+public class PublicCompanySpec {
+
+    @JsonProperty
+    private Jurisdiction jurisdiction;
+
+    @JsonIgnore
+    private String companyNumber;
```

Comment:
Don't we need to remove this from here?  We don't want people able to override the company number?